webauthn/webauthn.py

```diff
@@ -103,6 +105,13 @@ def __init__(self, challenge, rp_name, rp_id, user_id, username,
                              ', '.join(self._attestation_forms))
         self.attestation = attestation
 
+        if userVerification != None:
+            userVerification = str(userVerification).lower()
+            if userVerification not in self._userVerification:
+                raise ValueError('userVerification must be a string and one of ' +
+                                 ', '.join(self._userVerification))
+        self.userVerification = userVerification
```

Comment:
How about we put this line in an `else:` block? That way, it will `raise` if the provided value isn't one of the supported options, else set the value.

Reply:
Uniformity, see above

Reply:
👍